Api/.eslintrc.cjs

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
`.cjs`? Does this work?

Reply:
I noticed the `babel-eslint` didn't work as expected so I converted it to `eslintrc.json` file